lib/raven/configuration.rb

```diff
@@ -127,6 +127,12 @@ def initialize
       self.sanitize_fields = []
       self.sanitize_credit_cards = true
       self.environments = []
+
+      self.release = ENV['HEROKU_SLUG_COMMIT']
+
+      if self.release.nil? || self.release.empty?
+        self.release = `git rev-parse --short HEAD`.strip rescue nil
```

Comment:
Yup, that's right: if the user isn't on heroku, and isn't using git, then `release == nil`. Although not idea, I think this solution will auto-tag errors with commit in the majority of cases.